pom.xml

```diff
@@ -71,26 +71,4 @@
       <version>1.12.4-1</version>
     </dependency>
   </dependencies>
-
-  <repositories>
-    <repository>
-      <id>repo.jenkins-ci.org</id>
-      <url>https://repo.jenkins-ci.org/public/</url>
-    </repository>
-  </repositories>
-
-  <pluginRepositories>
-    <pluginRepository>
-      <id>repo.jenkins-ci.org</id>
-      <url>https://repo.jenkins-ci.org/public/</url>
-    </pluginRepository>
-  </pluginRepositories>
-
-    <distributionManagement>
-    <repository>
-      <id>repo.jenkins-ci.org</id>
```

Comment:
this is the issue as far as I can tell:
https://github.com/jenkinsci/plugin-pom/blob/3101645162a2c0a210fbd470140b7013fcd9799f/pom.xml#L875